src/test/resources/features/UpdateCoordinate.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Update Coordinate
+  In order to use the app
+  As a user
+  I want to update coordinate and check throw errors
+
+  Background:
+    Given There is a registered user with username "username" and password "password" and email "user@domain.com"
+    And I create a new valid Coordinate
+
+  Scenario: Update coordinate
+    Given I login as "username" with password "password"
+    When I update that Coordinate with new value "-89.99999,-179.99999"
+    Then The response code is 200
```

Comment:
It might be interesting to check that now, that updated coordinate, now stores the intended coordinates. You can do a get and then check that the returned JSON includes the expected values. Something similar to what is done in https://github.com/UdL-EPS-SoftArch/myroutes-api/blob/0f657e590a263cdb2e9cdddf11243a588e694178/src/test/java/cat/udl/eps/softarch/myroutes/steps/StepDefs.java#L69 or https://github.com/UdL-EPS-SoftArch/myroutes-api/blob/0f657e590a263cdb2e9cdddf11243a588e694178/src/test/java/cat/udl/eps/softarch/myroutes/steps/RegisterStepDefs.java#L100